lib/src/flagged/flagged_bloc.dart

```diff
@@ -49,9 +49,14 @@ import 'package:ox_coi/src/data/repository_stream_handler.dart';
 import 'flagged_events_state.dart';
 
 class FlaggedBloc extends Bloc<FlaggedEvent, FlaggedState> {
-  RepositoryEventStreamHandler repositoryStreamHandler;
+  RepositoryEventStreamHandler _repositoryStreamHandler;
   Repository<ChatMsg> _messageListRepository;
   bool _listenersRegistered = false;
+  int _chatId;
+
+  FlaggedBloc({int chatId}) {
+    _chatId = chatId;
+  }
```

Comment:
Is there a reason for this constructor?

Reply:
Keeping the `_chatId` private, this is the only way to do so, imho - isn't it?